stackslib/src/net/mod.rs

```diff
@@ -2135,6 +2135,25 @@ pub mod test {
             test_path
         }
 
+        /// Initialize the .miners StackerDB instance
+        pub fn init_stacker_db_miners(&mut self) {
```

Comment:
This shouldn't be necessary.  Instead, you can just call `peer.step_with_ibd(true)` in-between calls to `peer.next_burnchain_block()`. Then, the `.miners` contract DB state should initialize on its own.

Reply:
Ah kk ! I was wondering how ot enforce the stacker db miners being instantiated and this seemed the least janky option (though still janky). Will try with peer.step_with_ibd and see what happens!

Reply:
Actually I remember why I did this. Because it breaks test_make_miners_stackerdb_config as it attempts to build a block proposal which makes the config using the stacker db .miners contract which doesn't exist. Not sure how to fix this.